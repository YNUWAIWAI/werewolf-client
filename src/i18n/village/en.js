export default {
  'ChatContainer.date': 'Day {date}',
  'ChatContainer.postMortem': 'Post Mortem',
  'CommandInput.placeholder.limited': 'werewolf',
  'CommandInput.placeholder.postMortem': 'postMortem',
  'CommandInput.placeholder.private': 'private',
  'CommandInput.placeholder.public': 'public',
  'CommandInput.send': 'send'
}
