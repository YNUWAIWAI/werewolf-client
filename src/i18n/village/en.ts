export default {
  'ChatContainer.date': 'Day {date}',
  'ChatContainer.postMortem': 'Post-mortem discussion',
  'ChatDate': '{postTime}（{restTime} left）',
  'CommandInput.placeholder.limited': 'Werewolf',
  'CommandInput.placeholder.postMortem': 'Graveyard',
  'CommandInput.placeholder.private': 'Private',
  'CommandInput.placeholder.public': 'Public',
  'CommandInput.send': 'Send',
  'CommandNavigation.nextGame': 'Next Game',
  'CommandNavigation.returnToLobby': 'Return to the lobby',
  'CommandNavigation.showResult': 'Show results',
  'CommandSelection.Description.dayVote.fixed': 'This is the character you chose to put to death',
  'CommandSelection.Description.dayVote.unfixed': 'Choose a character to put to death',
  'CommandSelection.Description.hunterVote.fixed': 'This is the character you chose to protect',
  'CommandSelection.Description.hunterVote.unfixed': 'Choose a character to protect',
  'CommandSelection.Description.seerVote.fixed': 'This is the character you chose for your divination',
  'CommandSelection.Description.seerVote.unfixed': 'Choose a character for your divination',
  'CommandSelection.Description.wait': 'Please wait...',
  'CommandSelection.Description.werewolfVote.fixed': 'This is the character you chose to attack',
  'CommandSelection.Description.werewolfVote.unfixed': 'Choose a character to attack',
  'FormattedTime.end': 'Time\'s up',
  'FormattedTime.time': '{minute}\'{second} left',
  'Info.property.agent': 'You:',
  'Info.property.role': 'Role:',
  'Info.property.team': 'Team:',
  'InfoDay.day': 'Day {date}: noon',
  'InfoDay.morning': 'Day {date}: morning',
  'InfoDay.night': 'Day {date}: night',
  'InfoDay.postMortem': 'Post-mortem discussion',
  'InfoDay.result': 'Result',
  'Modal.Description.dayVote': 'Do you really want to put this character to death?',
  'Modal.Description.hunterVote': 'Do you really want to protect this character?',
  'Modal.Description.seerVote': 'Do you really want to choose this character for your divination?',
  'Modal.Description.wait': 'Please wait...',
  'Modal.Description.werewolfVote': 'Do you really want to attack this character?',
  'Modal.button.no': 'No',
  'Modal.button.yes': 'Yes',
  'Result.status(alive)': 'Alive',
  'Result.status(dead)': 'Dead',
  'Result.status(death by attack)': 'Killed in an attack',
  'Result.status(death by execution)': 'Executed',
  'Result.status(death by fear)': 'Scared to death',
  'Result.status(unnatural death)': 'Found dead',
  'Result.summary.description(audience)': 'Victory for Team {winnerTeam}',
  'Result.summary.description(player, lose)': 'Your Team {myTeam} lose. (Victory for Team {winnerTeam})',
  'Result.summary.description(player, win)': 'Your Team {myTeam} win. (Victory for Team {winnerTeam})',
  'Result.summary.loser(1)': 'Losers (Team {loserTeam0})',
  'Result.summary.loser(2)': 'Losers (Team {loserTeam0} and Team {loserTeam1})',
  'Result.summary.winner': 'Winners (Team {winnerTeam})',
  'TeamIcon.villager': 'Villagers',
  'TeamIcon.werehamster': 'Werewhamster',
  'TeamIcon.werewolf': 'Werewolves'
}
