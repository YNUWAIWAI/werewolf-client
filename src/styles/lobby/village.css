.lo--village {
  list-style-type: none;
  padding: 0;

  &--item {
    background: var(--color--background);
    border: solid 1px var(--color--border);
    display: grid;
    font-size: 2rem;
    grid-template-areas:
      'v-n-p v-n-v i-f-s'
      'h-n-p h-n-v h-n-v'
      's-p s-v1 s-v2'
      'mem mem mem'
      'c-p c-v c-v';
    grid-template-columns: minmax(200px, 1fr) minmax(200px, 1fr) minmax(200px, 1fr);
    margin-bottom: 0.5rem;

    &.build {
      grid-template-areas:
        'v-n-p v-n-v v-n-v'
        'h-n-p h-n-v h-n-v'
        's-p s-v1 s-v2'
        'mem mem mem'
        'c-p c-v c-v';
    }

    &.clickable {
      cursor: pointer;

      &:hover {
        background: var(--color--hover);

        & img {
          filter: var(--filter--hover);
        }

        &:active {
          background: var(--color--active);

          & img {
            filter: var(--filter--active);
          }
        }
      }
    }

    &--prop {
      align-items: center;
      border: solid 1px var(--color--border);
      box-sizing: border-box;
      display: inline-flex;
      justify-content: center;
      padding: 1rem;
      text-align: center;

      &.comment {
        grid-area: c-p;
      }

      &.host-name {
        grid-area: h-n-p;
      }

      &.setup {
        grid-area: s-p;
      }

      &.village-name {
        grid-area: v-n-p;
      }
    }

    &--val {
      align-items: center;
      border: solid 1px var(--color--border);
      box-sizing: border-box;
      display: inline-flex;
      font-size: inherit;
      justify-content: center;
      padding: 1rem;
      text-align: center;
      width: 100%;

      &.comment {
        grid-area: c-v;
        resize: vertical;
      }

      &.host-name {
        grid-area: h-n-v;
      }

      &.id-for-searching {
        grid-area: i-f-s;
      }

      &.setup1 {
        grid-area: s-v1;

        &.build {
          padding: 0;
        }
      }

      &--setup1--select {
        font-size: inherit;
        height: 100%;
        width: 100%;

        & > .lo--react-select {
          &__control {
            background: var(--color--background--input);
          }

          &__menu {
            background: var(--color--background--input--menu);
            z-index: var(--z-index--village--select);
          }
        }
      }

      &.setup2 {
        grid-area: s-v2;

        &.build {
          padding: 0;
        }
      }

      &--setup2--select {
        font-size: inherit;
        height: 100%;
        width: 100%;

        & > .lo--react-select {
          &__control {
            background: var(--color--background--input);
          }

          &__menu {
            background: var(--color--background--input--menu);
            z-index: var(--z-index--village--select);
          }
        }
      }

      &.village-name {
        grid-area: v-n-v;
      }

      input&, textarea& {
        background: var(--color--background--input);
        color: var(--color--font--input);
      }

      &.invalid {
        border-color: red;
      }

      &:invalid {
        color: red;
        outline-color: red;
        outline-offset: -2px;
      }
    }

    &--member {
      border: solid 1px var(--color--border);
      box-sizing: border-box;
      display: grid;
      grid-area: mem;
      grid-template-areas:
        'robot human'
        'role role';
      padding: 1rem;

      &--robot {
        align-items: center;
        border: solid 1px var(--color--border);
        display: inline-flex;
        font-size: inherit;
        grid-area: robot;
        justify-content: center;
        padding: 1rem;
        text-align: center;
      }

      &--human {
        align-items: center;
        border: solid 1px var(--color--border);
        display: inline-flex;
        font-size: inherit;
        grid-area: human;
        justify-content: center;
        padding: 1rem;
        text-align: center;
      }

      &--role {
        display: grid;
        grid-area: role;
        grid-template-columns: repeat(8, 1fr);
        margin-top: 0.5rem;

        &--item {
          display: grid;
          font-size: inherit;
          grid-template: 1fr calc(var(--size--role-image) - 16px) 16px 15px 1fr / 1fr calc(var(--size--role-image) - 24px) 12px 12px 1fr;

          &--caption {
            font-size: 10px;
            grid-column: 1 / 6;
            grid-row: 4;
            line-height: 1.5;
            text-align: center;
            word-break: keep-all;
          }

          &--image {
            grid-column: 2 / 4;
            grid-row: 2 / 4;
            height: var(--size--role-image);
            width: var(--size--role-image);
            z-index: var(--z-index--member--image);
          }

          &--number {
            color: var(--color--font--role-image);
            font-size: 1rem;
            grid-column: 4;
            grid-row: 3;
            line-height: 1;
            z-index: var(--z-index--member--text);
          }

          &--times {
            align-items: flex-end;
            color: var(--color--font--role-image);
            display: inline-flex;
            font-size: 0.75rem;
            grid-column: 3;
            grid-row: 3;
            justify-content: center;
            line-height: 1;
            z-index: var(--z-index--member--text);
          }
        }
      }
    }

    &--member-select {
      border: solid 1px var(--color--border);
      box-sizing: border-box;
      display: grid;
      grid-area: mem;
      grid-template-areas:
        'robot human'
        'A A'
        'B B'
        'C C';
      grid-template-columns: minmax(300px, 1fr) minmax(300px, 1fr);
      width: 100%;

      &--robot {
        align-items: center;
        border: solid 1px var(--color--border);
        display: inline-flex;
        font-size: inherit;
        grid-area: robot;
        justify-content: center;
        margin: 1rem 0 1rem 1rem;
        padding: 1rem;
        text-align: center;

        &--select {
          font-size: inherit;
          margin-left: 1rem;
          width: calc(100% - 4rem - 10px);

          & > .lo--react-select {
            &__control {
              background: var(--color--background--input);
              border-radius: 4px;
            }

            &__menu {
              background: var(--color--background--input--menu);
              z-index: var(--z-index--village--select);
            }
          }
        }

        &.invalid {
          border-color: red;
        }
      }

      &--human {
        align-items: center;
        border: solid 1px var(--color--border);
        display: inline-flex;
        font-size: inherit;
        grid-area: human;
        justify-content: center;
        margin: 1rem 1rem 1rem 0;
        padding: 1rem;
        text-align: center;
      }

      &--role {
        display: grid;
        grid-area: role;
        grid-template:
          'radio text list'
          / fit-content(1rem) fit-content(1rem) auto;
        margin: 0 4px 0 0;
        overflow: auto;
        padding: 8px 4px 8px 8px;

        &.A {
          grid-area: A;
        }

        &.B {
          grid-area: B;
        }

        &.C {
          grid-area: C;
        }

        &:hover {
          background: var(--color--hover);
        }

        &--list {
          display: grid;
          grid-area: list;
          grid-template-columns: repeat(8, minmax(67px, 1fr));

          &--item {
            display: grid;
            font-size: inherit;
            grid-template: 1fr calc(var(--size--role-image) - 16px) 16px 15px 1fr / 1fr calc(var(--size--role-image) - 24px) 12px 12px 1fr;

            &--caption {
              font-size: 10px;
              grid-column: 1 / 6;
              grid-row: 4;
              line-height: 1.5;
              text-align: center;
              word-break: keep-all;
            }

            &--image {
              grid-column: 2 / 4;
              grid-row: 2 / 4;
              height: var(--size--role-image);
              width: var(--size--role-image);
              z-index: var(--z-index--member--image);
            }

            &--number {
              color: var(--color--font--role-image);
              font-size: 1rem;
              grid-column: 4;
              grid-row: 3;
              line-height: 1;
              z-index: var(--z-index--member--text);
            }

            &--times {
              align-items: flex-end;
              color: var(--color--font--role-image);
              display: inline-flex;
              font-size: 0.75rem;
              grid-column: 3;
              grid-row: 3;
              justify-content: center;
              line-height: 1;
              z-index: var(--z-index--member--text);
            }
          }
        }

        &--radio {
          align-items: center;
          display: flex;
          grid-area: radio;
          justify-content: center;
        }

        &--text {
          align-items: center;
          display: flex;
          grid-area: text;
          justify-content: center;
          padding: 0.5rem;
          position: sticky;
        }
      }
    }

    &--transition {
      &-appear {
        opacity: 0;
      }

      &-appear-active {
        opacity: 1;
        transition: opacity 0.5s ease-out;
      }

      &-enter {
        opacity: 0;
      }

      &-enter-active {
        opacity: 1;
        transition: opacity 0.5s ease-out;
      }

      &-exit {
        opacity: 1;
      }

      &-exit-active {
        opacity: 0;
        transition: opacity 0.3s ease-in;
      }
    }
  }
}
