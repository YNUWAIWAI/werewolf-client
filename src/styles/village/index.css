@import './background';
@import './chat';
@import './command';
@import './grid';
@import './hideButton';
@import './info';
@import './keyframe';
@import './loader';
@import './modal';
@import './obfucator';
@import './prediction';
@import './result';
@import './theme';
