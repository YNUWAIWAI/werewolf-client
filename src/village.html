@(webSocketUrl: String)
@prefetch = {
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/hunter_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/madman_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/mason_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/medium_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/seer_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/werehamster_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/role_icons/50x50withTI/werewolf_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/anonymous_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/b_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/c_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/d_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/e_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/f_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/g_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/h_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/i_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/j_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/k_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/l_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/m_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/n_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/o_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/r_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/s_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/t_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/u_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/v_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/50x50/y_50x50.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/a_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/anonymous_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/b_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/c_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/d_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/e_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/f_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/g_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/h_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/i_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/j_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/k_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/l_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/m_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/n_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/o_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/r_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/s_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/t_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/u_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/v_120x120.png' as="image">
  <link rel="prefetch" href='https://werewolf.world/image/0.3/agent_icons/120x120/y_120x120.png' as="image">
}
@scripts = {
  <script crossorigin src="https://unpkg.com/fuse.js@@3"></script>
  <script crossorigin src="https://unpkg.com/prop-types@@15/prop-types.min.js"></script>
  <script crossorigin src="https://unpkg.com/react@@16/umd/react.production.min.js"></script>
  <script crossorigin src="https://unpkg.com/react-dom@@16/umd/react-dom.production.min.js"></script>
  <script crossorigin src="https://unpkg.com/react-intl@@2/dist/react-intl.min.js"></script>
  <script crossorigin src="https://unpkg.com/react-redux@@7/dist/react-redux.min.js"></script>
  <script crossorigin src="https://unpkg.com/redux@@4/dist/redux.min.js"></script>
  <script defer src='@routes.Assets.at("javascripts/village.js")' data-url='@webSocketUrl' id="script"></script>
}
@template("Village", "village.css", scripts, prefetch) {
  <div id="root"></div>
}
