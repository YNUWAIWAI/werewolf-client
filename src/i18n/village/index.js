// @flow
import type {Language} from 'village'
import en from './en'
import ja from './ja'

export const getMessages = (locale: Language) => {
  switch (locale) {
    case 'en':
      return en
    case 'ja':
      return ja
    default:
      throw Error(`${locale} is unsuported`)
  }
}
