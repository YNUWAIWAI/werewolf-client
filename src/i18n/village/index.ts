import * as village from '../../../types/village'
import en from './en.json'
import fr from './fr.json'
import it from './it.json'
import ja from './ja.json'

export const getMessages = (locale: village.Language) => {
  switch (locale) {
    case 'en':
      return en
    case 'fr':
      return fr
    case 'it':
      return it
    case 'ja':
      return ja
    default:
      throw Error(`${locale} is unsuported`)
  }
}
