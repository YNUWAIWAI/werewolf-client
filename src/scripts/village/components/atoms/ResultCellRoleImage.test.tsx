import * as React from 'react'
import * as village from 'types/village'
import ResultCellRoleImage from './ResultCellRoleImage'
import {shallow} from 'enzyme'

describe('<ResultCellRoleImage />', () => {
  test('alive', () => {
    const warpper = shallow(
      <ResultCellRoleImage
        image="https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png"
        name="Villager"
        status={village.AgentStatus.alive}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
  test('dead', () => {
    const warpper = shallow(
      <ResultCellRoleImage
        image="https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png"
        name="Villager"
        status={village.AgentStatus.dead}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
  test('deathByAttack', () => {
    const warpper = shallow(
      <ResultCellRoleImage
        image="https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png"
        name="Villager"
        status={village.AgentStatus.deathByAttack}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
  test('deathByExecution', () => {
    const warpper = shallow(
      <ResultCellRoleImage
        image="https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png"
        name="Villager"
        status={village.AgentStatus.deathByExecution}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
  test('deathByFear', () => {
    const warpper = shallow(
      <ResultCellRoleImage
        image="https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png"
        name="Villager"
        status={village.AgentStatus.deathByFear}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
  test('unnaturalDeath', () => {
    const warpper = shallow(
      <ResultCellRoleImage
        image="https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png"
        name="Villager"
        status={village.AgentStatus.unnaturalDeath}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
})
