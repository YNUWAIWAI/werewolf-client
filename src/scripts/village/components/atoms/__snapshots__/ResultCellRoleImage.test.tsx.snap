// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultCellRoleImage /> alive 1`] = `"<div class=\\"vi--result--cell--roleImage \\"><img class=\\"vi--result--cell--roleImage--image\\" src=\\"https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png\\"/><div class=\\"vi--result--cell--roleImage--name\\">Villager</div></div>"`;

exports[`<ResultCellRoleImage /> dead 1`] = `"<div class=\\"vi--result--cell--roleImage dead\\"><img class=\\"vi--result--cell--roleImage--image\\" src=\\"https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png\\"/><div class=\\"vi--result--cell--roleImage--name\\">Villager</div></div>"`;

exports[`<ResultCellRoleImage /> deathByAttack 1`] = `"<div class=\\"vi--result--cell--roleImage dead\\"><img class=\\"vi--result--cell--roleImage--image\\" src=\\"https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png\\"/><div class=\\"vi--result--cell--roleImage--name\\">Villager</div></div>"`;

exports[`<ResultCellRoleImage /> deathByExecution 1`] = `"<div class=\\"vi--result--cell--roleImage dead\\"><img class=\\"vi--result--cell--roleImage--image\\" src=\\"https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png\\"/><div class=\\"vi--result--cell--roleImage--name\\">Villager</div></div>"`;

exports[`<ResultCellRoleImage /> deathByFear 1`] = `"<div class=\\"vi--result--cell--roleImage dead\\"><img class=\\"vi--result--cell--roleImage--image\\" src=\\"https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png\\"/><div class=\\"vi--result--cell--roleImage--name\\">Villager</div></div>"`;

exports[`<ResultCellRoleImage /> unnaturalDeath 1`] = `"<div class=\\"vi--result--cell--roleImage dead\\"><img class=\\"vi--result--cell--roleImage--image\\" src=\\"https://werewolf.world/image/0.3/role_icons/50x50withTI/villager_50x50.png\\"/><div class=\\"vi--result--cell--roleImage--name\\">Villager</div></div>"`;
